orchestrator-salt/src/main/resources/salt/salt/postgresql/disaster_recovery/scripts/restore_db.sh

```diff
@@ -3,7 +3,6 @@
 # This script uses the 'psql' cli to drop hive and ranger databases, create them, then read in a plain SQL file to restore data.
 # We retrieve the SQL files from a valid URL, which should be `s3a://` or `abfs://` for AWS or Azure clouds respectively.
 
-set -o errexit
```

Comment:
Was this the reason why the error was not logged?

Reply:
Yes.